src/Common/memory.h

```diff
@@ -13,7 +13,11 @@
 #    include <jemalloc/jemalloc.h>
 #endif
 
-#if !USE_JEMALLOC
+#if USE_MIMALLOC
```

Comment:
Debugging stuff ok, but I guess it will be enough to handle mimalloc first and jemalloc as a fallback